src/averageAges.js

```diff
@@ -37,7 +42,16 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = withChildren
+    ? people.filter((p) => p.sex === 'f'
+      && people.find((p1) => p1.mother === p.name))
+    : people.filter((p) => p.sex === 'f');
```

Comment:
Use a descriptive value for properties, one suggestion would be to use a const to store values that are repeated. This makes it easier to understand and maintain. For example:
const MALE = "m"
const FEMALE = "f"

people.filter(p => p.sex === FEMALE